<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinUnblock</class>
 <widget class="QDialog" name="PinUnblock">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>570</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font-family: Roboto, Helvetica;
font-size: 14px;
}
#PinUnblock {
background-color: #FFFFFF;
border-radius: 4px;
}
#label {
color: #003168;
font-size: 20px;
font-weight: 700;
}
#labelPuk, #labelPin, #labelRepeat {
color: #07142A;
}
#errorPuk, #errorPin, #errorRepeat {
color: #AD2A45;
}
QLineEdit {
padding: 10px 14px;
border: 1px solid #C4CBD8;
border-radius: 4px;
color: #07142A;
placeholder-text-color: #607496;
font-size: 16px;
}
QPushButton {
padding: 12px 12px;
border-radius: 4px;
border: 1px solid #AD2A45;
color: #AD2A45;
font-weight: 700;
}
QPushButton:hover {
background-color: #F5EBED;
}
QPushButton:pressed {
background-color: #E1C1C6;
}
QPushButton:default {
color: #ffffff;
border-color: #2F70B6;
background-color: #2F70B6;
}
QPushButton:default:hover {
background-color: #2B66A6;
}
QPushButton:default:pressed {
background-color: #215081;
}
QPushButton:default:disabled {
background-color: #2F70B6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="PinUnblockLayout">
   <property name="spacing">
    <number>40</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">%1 code change</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutContent">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <layout class="QGridLayout" name="infoBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="line1_bullet">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">&amp;bull;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="line1_text">
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::TabFocus</enum>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="line2_bullet">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">&amp;bull;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="line2_text">
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::TabFocus</enum>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="line3_bullet">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">&amp;bull;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="line3_text">
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::TabFocus</enum>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="inputAndLabelPukLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPuk">
         <property name="text">
          <string>Current PUK code</string>
         </property>
         <property name="buddy">
          <cstring>puk</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="puk">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorPuk"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="inputAndLabelPinLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPin">
         <property name="text">
          <string>New %1 code</string>
         </property>
         <property name="buddy">
          <cstring>pin</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pin">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorPin"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="inputAndLabelPinRepeatLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labelRepeat">
         <property name="text">
          <string>Repeat new %1 code</string>
         </property>
         <property name="buddy">
          <cstring>repeat</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="repeat">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorRepeat"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="change">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Unblock</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
